hyppo/ksample/tests/test_ksamp.py

```diff
@@ -45,6 +45,13 @@ def test_rep(self, n, obs_stat, obs_pvalue, indep_test):
         assert stat == stat2
         assert pvalue == pvalue2
 
+    def test_energy(self):
```

Comment:
Energy already has it's own testing class. See https://github.com/neurodata/hyppo/blob/main/hyppo/ksample/tests/test_energy.py